python.html.markdown

```diff
@@ -347,6 +347,18 @@ for i in range(4):
     print(i)
 
 """
+"range(lower, upper)" returns a list of numbers
+from the lower number to the upper number
+prints:
+    4
+    5
+    6
+    7
+"""
+for i in range(4, 8):
+    print(i)
```

Comment:
The whole doc uses the print_function from future. This isn't a good way to teach/learn python2 in my opinion, but I was just sticking to the format of the existing doc.